src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -129,7 +135,7 @@
       text-decoration: none;
 
       &:focus {
-        outline: 2px solid #f9c642;
+        outline: 2px solid $color-gold-light;
```

Comment:
https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L85